homeassistant/components/ssdp/__init__.py

```diff
@@ -205,16 +205,30 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     return True
 
 
-async def _async_process_callbacks(
+async def _async_run_ssdp_callback(
```

Comment:
we have a helper for this too `async_create_catching_coro`

Reply:
I went to look at it and the first thing I saw was :
https://github.com/home-assistant/core/blob/869128e951dd4fb93392032ef39660d1d36e1e4b/homeassistant/util/logging.py#L201